Queries (#3)
temporalio/edu-interacting-with-workflows-typescript: querying-your-workflows/exercise-querying-workflows.md

# Hands-On Exercise #3: Querying Workflows

During this exercise, you will:

- Define and handle a Query

> - Get a handle on the Workflow the developer would like to query

Author: Did not know that! Thanks and fixed

> - Create a handle to the Workflow the developer would like to query

Reviewer: ```suggestion
- Create a handle to Workflow to be queried
```
